large_language_model_pretraining/nemo/callbacks.py

```diff
@@ -202,12 +202,12 @@ def on_train_end(self, trainer, pl_module):
     
     @rank_zero_only
     def on_validation_start(self, trainer, pl_module):
-        mllogger.end(key=constants.BLOCK_STOP, metadata={'epoch_num': self.consumed_tokens(trainer)})
-        mllogger.start(key=constants.EVAL_START, metadata={'epoch_num': self.consumed_tokens(trainer)})
+        mllogger.end(key=constants.BLOCK_STOP, metadata={constants.SAMPLES_COUNT: self.consumed_tokens(trainer)})
```

Comment:
not sure where `self.consumed_tokens()` is defined, but does it return tokens or samples? If tokens, we need to switch it to samples. If samples, we need to rename it